README.md

```diff
@@ -22,37 +22,66 @@ gitbook install
 
 ## Options
 
-You can put your template into `book.js`(`book.json`) by `template` option.
+Several options can be set in `book.json` to customize the plugin.
+
+| option | value | Description |
+| --- | --- | --- |
+| `template` | `{default,full,ace,...}` or custom path | reindent code if marker or slice is used |
+| `unindent` | `{true,false}` | reindent code if marker or slice is used |
+| `edit` | `{true,false}` | allow edit code (**ace template required**) |
+| `check` | `{true,false}` | check syntax (**ace template required**) |
+| `theme` | `{monokai,coffee,...}` | check syntax (**ace template required**) |
```

Comment:
ok 👍 (use same tags than ace plugin)
`check` verify the lang syntax
I'll fix that